doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -74,8 +75,21 @@ lose their role as parameters. For example:
 
    Notation "'IF' c1 'then' c2 'else' c3" := (c1 /\ c2 \/ ~ c1 /\ c3) (at level 200, right associativity).
 
-Symbols that start with a single quote with 3 or more characters must be single quoted.
-For example, the symbol `'ab` is represented by `''ab'` in the notation string.
+Symbols that start with a single quote and followed by at least 2
+characters must be single quoted.  For example, the symbol `'ab` is
+represented by `''ab'` in the notation string. Strings can be used in
+notations: the start and end of a string symbol in a notation string
+must be doubled double quotes. Spaces occurring in these strings are
+part of the string symbol and do not contribute to the separation
+between tokens. To refer to double quotes in these strings, use four
```

Comment:
```suggestion
notations: they must begin and end with two double quotes.
Embedded spaces in these strings are
part of the string and do not contribute to the separation
between notation tokens. To embed double quotes in these strings, use four
```

Reply:
Are you going to take any of my suggestions here, or an oversight?  I thought "doubled double quote" was pretty awkward.

Reply:
The suggestions were overlapping and I got a github error which apparently resulted in losing the above. My intent was to take the suggestion.